stdlib/public/stubs/DispatchShims.mm

```diff
@@ -104,6 +112,15 @@
 }
 
 void
+swift::_swift_dispatch_group_notify(
+	__swift_shims_dispatch_group_t group,
+	__swift_shims_dispatch_queue_t queue,
+	__swift_shims_dispatch_block_t block)
+{
+	dispatch_group_notify((dispatch_group_t)group, cast(queue), cast(block));
```

Comment:
Could you add `dispatch_group_t` to the `BRIDGE_TYPE` macros above?  This will allow us to use the `cast()` function.